mmv1/third_party/terraform/services/netapp/resource_netapp_active_directory_test.go

```diff
@@ -0,0 +1,142 @@
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNetappactiveDirectory_activeDirectoryCreateExample_Basic(context),
+			},
+			{
+				ResourceName:            "google_netapp_active_directory.test_active_directory_basic",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"location", "active_directory_id", "pass", "labels", "terraform_labels"},
+			},
+			{
+				Config: testAccNetappactiveDirectory_activeDirectoryCreateExample_Full(context),
+			},
+			{
+				ResourceName:            "google_netapp_active_directory.test_active_directory_full",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"location", "active_directory_id", "pass", "labels", "terraform_labels"},
+			},
+			{
+				Config: testAccNetappactiveDirectory_activeDirectoryCreateExample_Update(context),
+			},
+			{
+				ResourceName:            "google_netapp_active_directory.test_active_directory_full",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"location", "active_directory_id", "pass", "labels", "terraform_labels"},
+			},
+		},
+	})
+}
+
+func testAccNetappactiveDirectory_activeDirectoryCreateExample_Basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_netapp_active_directory" "test_active_directory_basic" {
+    active_directory_id = "tf-test-test-active-directory-basic%{random_suffix}"
+	location = "us-central1"
+	domain = "ad.internal"
+	dns = "172.30.64.3"
+	net_bios_prefix = "smbserver"
+	username = "user"
+	password = "pass"
+  }
+`, context)
+}
+
+func testAccNetappactiveDirectory_activeDirectoryCreateExample_Full(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_netapp_active_directory" "test_active_directory_full" {
+    active_directory_id = "tf-test-test-active-directory-full%{random_suffix}"
+    location = "us-central1"
+    domain = "ad.internal"
+    dns = "172.30.64.3"
+    net_bios_prefix = "smbserver"
+    username = "user"
+    password = "pass"
+    aes_encryption         = false
+    backup_operators       = "administrator,cvs-svc,nfsuser"
+    description            = "ActiveDirectory is the public representation of the active directory config."
+    encrypt_dc_connections = false
+    kdc_hostname           = "hostname"
+    kdc_ip                 = "10.10.0.11"
+    labels                 = { 
+        "name": "wrench", 
+        "mass": "1.3kg", 
+        "count": "3" 
+    }
+    ldap_signing           = false
+    nfs_users_with_ldap    = false
+    organizational_unit    = "CN=Computers"
+    security_operators     = "administrator,cvs-svc,nfsuser"
+    site                   = "test-site"
+  }
+`, context)
+}
+
+func testAccNetappactiveDirectory_activeDirectoryCreateExample_update(context map[string]interface{}) string {
```

Comment:
```suggestion
func testAccNetappactiveDirectory_activeDirectoryCreateExample_Update(context map[string]interface{}) string {
```

this is causing an error! sorry for mislabelling my lines